core.lua

```diff
@@ -1935,6 +2052,18 @@ do
 					-- Update game tooltip with player info
 					ShowTooltip(tooltip, bor(TooltipProfileOutput.PADDING(), ProfileOutput.ADD_LFD), leaderName, nil, PLAYER_FACTION, true, LFD_ACTIVITYID_TO_DUNGEONID[activityID], keystoneLevel)
 					ns.PROFILE_UI.ShowProfile(leaderName, nil, PLAYER_FACTION, tooltip, nil, activityID, keystoneLevel)
+
+					if ns.addonConfig.enableLeaderboardDataInTooltips then
+						local index = LFD_ACTIVITYID_TO_DUNGEONID[activityID]
+						if index then
+							local dungeon = CONST_DUNGEONS[index]
+
+							tooltip:AddLine(" ")
```

Comment:
Wondering (but this would be more for @Vladinator89 ) if it wouldn't be better to plug this with the `ShowTooltip` and the different tag that exists in it (issue though is to add all the optionnal data, such as the leader name, and so on).

Maybe the best is to do this PR (and merge it), and maybe later see (either you if you want, vlad or myself) if it's possible to use this `ShowTooltip` function :)